Emby.Server.Implementations/Cryptography/CryptographyProvider.cs

```diff
@@ -125,37 +123,27 @@ public byte[] ComputeHash(string hashMethod, byte[] bytes, byte[] salt)
                     }
                 }
             }
-            else
-            {
-                throw new CryptographicException($"Requested hash method is not supported: {hashMethod}");
-            }
+
+            throw new CryptographicException($"Requested hash method is not supported: {hashMethod}");
+
         }
 
         public byte[] ComputeHashWithDefaultMethod(byte[] bytes, byte[] salt)
-        {
-            return PBKDF2(DefaultHashMethod, bytes, salt, _defaultIterations);
-        }
+            => PBKDF2(DefaultHashMethod, bytes, salt, _defaultIterations);
 
         public byte[] ComputeHash(PasswordHash hash)
         {
             int iterations = _defaultIterations;
             if (!hash.Parameters.ContainsKey("iterations"))
             {
-                hash.Parameters.Add("iterations", _defaultIterations.ToString(CultureInfo.InvariantCulture));
+                hash.Parameters.Add("iterations", iterations.ToString(CultureInfo.InvariantCulture));
```

Comment:
Why are these parameters strings in a dictionary anyway? Is that just legacy stuff?

Reply:
https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Model/Cryptography/PasswordHash.cs#L11
Because they are just key-value pairs.

Reply:
I suppose I would have expected a class or struct.